error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (C:\\Users\\91809\\Desktop\\bookstore\\src\\config\\/database.js:12:20)\n    at tryCatch (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:32:7\n    at new Promise (<anonymous>)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:21:12","timestamp":"2022-07-04T10:54:14.949Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (C:\\Users\\91809\\Desktop\\bookstore\\src\\config\\/database.js:12:20)\n    at tryCatch (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:32:7\n    at new Promise (<anonymous>)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:21:12","timestamp":"2022-07-04T11:06:23.116Z"}
error: Could not connect to the database. connect ECONNREFUSED ::1:27017 {"reason":{"commonWireVersion":null,"compatibilityError":null,"compatible":true,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"logicalSessionTimeoutMinutes":null,"maxElectionId":null,"maxSetVersion":null,"servers":{},"setName":null,"stale":false,"type":"Single"},"stack":"MongooseServerSelectionError: connect ECONNREFUSED ::1:27017\n    at NativeConnection.Connection.openUri (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\connection.js:847:32)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:32:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at _callee$ (C:\\Users\\91809\\Desktop\\bookstore\\src\\config\\/database.js:12:20)\n    at tryCatch (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:86:17)\n    at Generator._invoke (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:66:24)\n    at Generator.next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\regeneratorRuntime.js:117:21)\n    at asyncGeneratorStep (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:3:24)\n    at _next (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:25:9)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:32:7\n    at new Promise (<anonymous>)\n    at C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\runtime\\helpers\\asyncToGenerator.js:21:12","timestamp":"2022-07-04T11:14:38.538Z"}
error: 
    status - 500 
    message - ValidationError: "firstName" is required 
    url - /api/v1/users/newuser 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-04T14:32:45.000Z"}
error: 
    status - 500 
    message - ValidationError: "cofirmPassword" is required 
    url - /api/v1/users/newuser 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-04T14:35:40.034Z"}
error: uncaughtException: bookRoute is not defined
ReferenceError: bookRoute is not defined
    at routes (C:\Users\91809\Desktop\bookstore\src\routes\/index.js:15:24)
    at Object.<anonymous> (C:\Users\91809\Desktop\bookstore\src\/index.js:32:32)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Module._compile (C:\Users\91809\Desktop\bookstore\node_modules\pirates\lib\index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Object.newLoader [as .js] (C:\Users\91809\Desktop\bookstore\node_modules\pirates\lib\index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:827:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at Object.<anonymous> (C:\Users\91809\Desktop\bookstore\node_modules\@babel\node\lib\_babel-node.js:176:21)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:827:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47 {"date":"Mon Jul 04 2022 22:40:13 GMT+0530 (India Standard Time)","error":{},"exception":true,"os":{"loadavg":[0,0,0],"uptime":35043},"process":{"argv":["node","C:\\Users\\91809\\Desktop\\bookstore\\src\\index.js"],"cwd":"C:\\Users\\91809\\Desktop\\bookstore","execPath":"C:\\Program Files\\nodejs\\node.exe","gid":null,"memoryUsage":{"arrayBuffers":18849307,"external":20300425,"heapTotal":78974976,"heapUsed":51384800,"rss":111648768},"pid":5984,"uid":null,"version":"v18.2.0"},"stack":"ReferenceError: bookRoute is not defined\n    at routes (C:\\Users\\91809\\Desktop\\bookstore\\src\\routes\\/index.js:15:24)\n    at Object.<anonymous> (C:\\Users\\91809\\Desktop\\bookstore\\src\\/index.js:32:32)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Module._compile (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\pirates\\lib\\index.js:136:24)\n    at Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Object.newLoader [as .js] (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\pirates\\lib\\index.js:141:7)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:827:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at Object.<anonymous> (C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\node\\lib\\_babel-node.js:176:21)\n    at Module._compile (node:internal/modules/cjs/loader:1105:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)\n    at Module.load (node:internal/modules/cjs/loader:981:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:827:12)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)\n    at node:internal/main/run_main_module:17:47","timestamp":"2022-07-04T17:10:13.145Z","trace":[{"column":24,"file":"C:\\Users\\91809\\Desktop\\bookstore\\src\\routes\\/index.js","function":"routes","line":15,"method":null,"native":false},{"column":32,"file":"C:\\Users\\91809\\Desktop\\bookstore\\src\\/index.js","function":null,"line":32,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":24,"file":"C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Module._compile","line":136,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":7,"file":"C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\pirates\\lib\\index.js","function":"Object.newLoader [as .js]","line":141,"method":"js]","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":827,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":21,"file":"C:\\Users\\91809\\Desktop\\bookstore\\node_modules\\@babel\\node\\lib\\_babel-node.js","function":null,"line":176,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1105,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1159,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":981,"method":"load","native":false},{"column":12,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":827,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":77,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":17,"method":null,"native":false}]}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:23:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:27:14.068Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:23:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:39:22.705Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:23:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:39:39.735Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:23:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:48:19.586Z"}
error: 
      status - 400
      message - Authorization token is required 
      url - /api/v1/books/getbyId/6214803b05253301a4fda584 
      method - GET 
      IP - ::1
     {"timestamp":"2022-07-05T08:48:43.990Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:23:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:49:05.937Z"}
error: 
    status - 500 
    message - TypeError: Cannot read properties of undefined (reading 'email')
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:24:28)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:50:38.893Z"}
error: 
    status - 500 
    message - ReferenceError: user is not defined
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:24:23)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:52:25.264Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:25:17)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T08:56:34.017Z"}
error: 
    status - 500 
    message - ReferenceError: UserID is not defined
    at _callee$ (C:\Users\91809\Desktop\bookstore\src\middlewares\/auth.middleware.js:25:17)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/books/getbyId/6214803b05253301a4fda584 
    method - GET 
    IP - ::1
   {"timestamp":"2022-07-05T09:06:28.744Z"}
error: 
    status - 500 
    message - Error: User doesnt exist
    at _callee2$ (C:\Users\91809\Desktop\bookstore\src\services\/user.service.js:12:11)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-05T09:21:33.182Z"}
error: 
    status - 500 
    message - Error: User doesnt exist
    at _callee2$ (C:\Users\91809\Desktop\bookstore\src\services\/user.service.js:12:11)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-05T09:22:08.249Z"}
error: 
    status - 500 
    message - Error: User doesnt exist
    at _callee2$ (C:\Users\91809\Desktop\bookstore\src\services\/user.service.js:12:11)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-05T09:24:10.323Z"}
error: 
    status - 500 
    message - Error: User doesnt exist
    at _callee2$ (C:\Users\91809\Desktop\bookstore\src\services\/user.service.js:12:11)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-05T09:25:23.470Z"}
error: 
    status - 500 
    message - Error: User doesnt exist
    at _callee2$ (C:\Users\91809\Desktop\bookstore\src\services\/user.service.js:12:11)
    at tryCatch (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:86:17)
    at Generator._invoke (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:66:24)
    at Generator.next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\regeneratorRuntime.js:117:21)
    at asyncGeneratorStep (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
    at _next (C:\Users\91809\Desktop\bookstore\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
    at processTicksAndRejections (node:internal/process/task_queues:95:5) 
    url - /api/v1/users/login 
    method - POST 
    IP - ::1
   {"timestamp":"2022-07-05T09:27:40.592Z"}
